portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,8 +75,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /**
-  * Gets comments from data store.
-  */
+   * Gets comments from data store.
+   */
   private List<Comment> getDataStoreComments(PreparedQuery results) {
```

Comment:
Can the `results` parameter be explained a bit better? I understand that the method is private.